### ISSUE TITLE
Dweomer Steel

### ISSUE THREAD

**Issue by Blackenx1** [LegionDark/Issues#1638]
Dweomer Steel Synth combination doesn't work

**Date & Time**: 
8:24pm Est 9/26/2017

**FFXI Client Version (use `/ver`)**: 

30170728_2
**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: no 3017070

**Character Name**: Blacken

**Nation**: Sandy

**Job(level)/Sub Job(level)**: 
Blu 86/43

**NPC or Monster or item Name**: 
Dweomer Steel

**Zone name**: Metalworks

**Coordinates (use `$where`)**: 

**ffxiah.com link (for items issues only)**: 
http://www.ffxiah.com/item/2764/dweomer-steel

**Multi-boxing? (multiple clients on same connection)**:  no

**Steps To Reproduce / any other info**: 
Basically the synth combination doesn't work. Fire crystal, 3 iron ore, 1 swamp ore

**Comment by R3P0FFXI** [comment]
@LegionDark/gm please submit the correct synth combination to DSP

**Comment by Raelious** [comment]
Fixed
